optimum/onnxruntime/modeling_seq2seq.py

```diff
@@ -601,6 +633,89 @@ def _from_pretrained(
     ):
         model_path = Path(model_id)
 
+        decoder_merged_path = None
+        if use_merged is not False:
```

Comment:
Maybe specify that `is not False` does not mean `is True` since it can also be `None`.